cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1032,7 +1133,8 @@ impl pallet_nfts::Config for Runtime {
 	type WeightInfo = weights::pallet_nfts::WeightInfo<Runtime>;
 	#[cfg(feature = "runtime-benchmarks")]
 	type Helper = ();
-	type BlockNumberProvider = frame_system::Pallet<Runtime>;
+	// TODO migrate for async backing.
+	type BlockNumberProvider = System;
```

Comment:
Maybe we still need to do this

Reply:
what you mean? storage migration needed?